workspaces/src/types/account.rs

```diff
@@ -30,7 +30,7 @@ impl fmt::Debug for Account {
 impl Account {
     /// Create a new account with the given path to the credentials JSON file
     pub fn from_file(
-        path: impl AsRef<std::path::Path>,
```

Comment:
Note that this was redundant to have as it is already imported, so I had safely removed it.